config/settings.yml

```diff
@@ -400,7 +400,8 @@ intent_to_file:
   prefill: true
 
 ivc_champva:
-  prefill: true
+  pega_api:
+    api_key: fake_api_key
```

Comment:
I think we want that prefill line in here (for when users have save-in-progress'd data).

Reply:
Also, I've added the production API key to the AWS param store under this path.

Reply:
Thanks!  I wasn't sure if that prefill line was a placeholder or if it was doing something.